modules/page_object_generic_pdf.py

```diff
@@ -0,0 +1,12 @@
+from modules.page_base import BasePage
+
+
+class GenericPdf(BasePage):
+    """
+    Generic POM for any page with an open PDF in it.
+    """
+
+    URL_TEMPLATE = "{pdf_url}"
```

Comment:
is this matched during runtime to a regex or just a placeholder? just wondering

Reply:
The `{pdf_url}` is replaced with the url passed to the POM, much like how we choose our category in `AboutPrefs`. Just lets you use it to open any PDF, and the POM should only contain things that are general PDF markup elements.